plugins/source/aws/resources/services/account/alternate_contacts.go

```diff
@@ -12,7 +12,7 @@ func AlternateContacts() *schema.Table {
 		Name:        "aws_account_alternate_contacts",
 		Description: `https://docs.aws.amazon.com/accounts/latest/reference/API_AlternateContact.html`,
 		Resolver:    fetchAccountAlternateContacts,
-		Multiplex:   client.AccountMultiplex,
+		Multiplex:   client.ServiceAccountRegionMultiplexer("account"),
```

Comment:
I'd have to check the most recent code, and I may very well be missing something, but  `client.AccountMultiplex` (multiplex over all accounts) seems quite different from `client.ServiceAccountRegionMultiplexer("account")` (multiplex over all accounts and regions for a service called `"account"`) - is this definitely right?

Reply:
It is only correct because in actuality there is only a single region (per partition) that supports this service

Reply:
[Here is how the `account` service is represented in that file now](https://github.com/erezrokah/cloudquery/blob/9397419590b9724d1ed5bd9743579ae742ec639f/plugins/source/aws/client/data/partition_service_region.json#L43-L47)

Reply:
Ah cool, I see, thanks for the explanation!